feat(aps): add monthly commitment (#4601)
scaleway/docs-content: pages/apple-silicon/how-to/manage-commitment-plan.mdx

---
meta:

>   title: How to manage the Commitment Plan for a Mac mini

Reviewer: ```suggestion
  title: How to manage the commitment plan for a Mac mini
```

>   description: This page explains how to manage the Commitment Plan for Scaleway's Mac mini servers.

Reviewer: ```suggestion
  description: This page explains how to manage the commitment plan for Scaleway's Mac mini servers.
```

content:

>   h1: How to manage the Commitment Plan for a Mac mini

Reviewer: ```suggestion
  h1: How to manage the commitment plan for a Mac mini
```

>   paragraph: This page explains how to manage the Commitment Plan for Scaleway's Mac mini servers.

Reviewer: ```suggestion
  paragraph: This page explains how to manage the commitment plan for Scaleway's Mac mini servers.
```

tags: apple-silicon, mac-mini, commitment-plan
dates:
  validation: 2025-03-12
  posted: 2025-03-12
categories:
  - bare-metal
---

> **Commitment Plan for Apple silicon** is a flexible pricing option that allows you to choose between a fully flexible pay-as-you-go pricing or a long-term monthly commitment period for their Apple silicon servers.

Reviewer: ```suggestion

**Commitment plan for Apple silicon** is a flexible pricing option that allows you to choose between a fully flexible pay-as-you-go pricing or a long-term monthly commitment period for their Apple silicon servers.
```

> By committing to a plan, you can enjoy better rates compared to pay-as-you-go pricing.

Reviewer: ```suggestion
By committing to a plan, you can enjoy better rates than pay-as-you-go pricing.
```

> The initial commitment is for one month from the order of the server. At the end of your initial commitment period, the commitment is automatically renewed every month on the anniversary date, unless you choose to delete the machine, in which case the deletion will take effect on the anniversary date of the commitment. 

Reviewer: ```suggestion
The initial commitment is for one month from the server order. At the end of your initial commitment period, the commitment is automatically renewed every month on the anniversary date unless you choose to delete the machine, in which case the deletion will take effect on the anniversary date of the commitment.
```

> It is important to note that the commitment concerns the server and it's associated options (such as the VPC option), which also have lower prices with monthly billing.

Reviewer: ```suggestion
It is important to note that the commitment concerns the server and its associated options, such as VPC, which also have lower prices with monthly billing.
```

<Message type="note">
  Currently, it is not possible to switch from a monthly commitment to hourly billing.
</Message>

> 

Reviewer: ```suggestion
```

<Macro id="requirements" />
- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
- A [Mac mini](/apple-silicon/how-to/create-mac-mini/)

> 1. Click **Apple silicon** in the **Bare Metal** section of the side menu. A list of your Mac minis displays.

Reviewer: ```suggestion
1. Click [Apple silicon](https://console.scaleway.com/asaas/servers) in the **Bare Metal** section of the side menu. A list of your Mac minis displays.
```

2. Click the Mac mini you want to configure. Your Mac mini information page displays.
3. Scroll down to the **Commitment plan** section. You can see information about your current commitment plan. Click **Manage plan**. A pop-up displays.

> 4. Select the commitment plan for Mac mini. Two options are available:

Reviewer: ```suggestion
4. Select the commitment plan for your Mac mini. Two options are available:
```

    - No commitment: You can terminate the subscription of your Mac mini at any time after the initial 24 hours, which is a mandatory period required by Apple's software license. [Learn more about Apple's software license requirements (Section 3.A)](https://www.apple.com/legal/sla/docs/macOSSequoia.pdf).

>     - Commitment: You subscribe to a [Commitment Plan for Apple silicon](/apple-silicon/concepts/#commitment-plan) and benefit from a discounted monthly price depending on your commitment. The commitment period can be chosen at the time of creation or later, but it is automatically renewed every month on the anniversary date.

Reviewer: ```suggestion
    - Commitment: You subscribe to a [commitment plan for Apple silicon](/apple-silicon/concepts/#commitment-plan) and benefit from a discounted monthly price based on your choice. The commitment period can be defined at the time of creation or later, but it is automatically renewed every month on the anniversary date.
```

    <Message type="important">

>       It is currently not possible to switch from a monthly commitment to hourly billing.

Reviewer: ```suggestion
      Currently, it is not possible to switch from a monthly commitment to hourly billing.
```

    </Message>
    - Click **Review** to visualize a summary of your choice.
5. Verify your new commitment plan and click **Update plan** to confirm.
    <Message type="note">

>         - Changes to your commitment plan will only take effect after your current commitment period has ended.

Reviewer: ```suggestion
        - Changes to your commitment plan will only take effect after your current cycle has ended.
```
